### ISSUE TITLE
Dependencies Screen error

### ISSUE THREAD

**Issue by stroomdev10** [gchq/stroom#2888]
[Dependencies.txt](https://github.com/gchq/stroom/files/8778557/Dependencies.txt)

**Comment by at055612** [comment]
inlining stack.  
```
Error getting dependencies for criteria DependencyCriteria{partialName='tot'} 
java.lang.IllegalArgumentException: Illegal Capacity: -96
	at java.base/java.util.ArrayList.&lt;init&gt;(ArrayList.java:160)
	at stroom.util.shared.ResultPage.createPageLimitedList(ResultPage.java:93)
	at stroom.importexport.impl.DependencyServiceImpl.getDependencies(DependencyServiceImpl.java:128)
	at stroom.importexport.impl.DependencyServiceImpl.lambda$getDependencies$1(DependencyServiceImpl.java:103)
	at stroom.task.impl.TaskContextFactoryImpl.lambda$wrap$2(TaskContextFactoryImpl.java:255)
	at stroom.util.logging.LocationAwareLambdaLogger.logDurationIfDebugEnabled(LocationAwareLambdaLogger.java:156)
	at stroom.task.impl.TaskContextFactoryImpl.lambda$wrap$4(TaskContextFactoryImpl.java:255)
	at stroom.util.pipeline.scope.PipelineScopeRunnable.scopeResult(PipelineScopeRunnable.java:38)
	at stroom.task.impl.TaskContextFactoryImpl.lambda$wrap$5(TaskContextFactoryImpl.java:252)
	at stroom.security.impl.SecurityContextImpl.asUserResult(SecurityContextImpl.java:232)
	at stroom.task.impl.TaskContextFactoryImpl.lambda$wrap$9(TaskContextFactoryImpl.java:252)
	at stroom.importexport.impl.DependencyServiceImpl.getDependencies(DependencyServiceImpl.java:109)
	at stroom.importexport.impl.ContentServiceImpl.lambda$fetchDependencies$3(ContentServiceImpl.java:122)
	at stroom.security.impl.SecurityContextImpl.secureResult(SecurityContextImpl.java:448)
	at stroom.importexport.impl.ContentServiceImpl.fetchDependencies(ContentServiceImpl.java:122)
	at stroom.importexport.impl.ContentResourceImpl.lambda$fetchDependencies$6(ContentResourceImpl.java:183)
	at event.logging.impl.EventLoggerBuilderImpl.loggedResult(EventLoggerBuilderImpl.java:146)
	at event.logging.impl.EventLoggerBuilderImpl.access$600(EventLoggerBuilderImpl.java:24)
	at event.logging.impl.EventLoggerBuilderImpl$ResultSubBuilderImpl.getResultAndLog(EventLoggerBuilderImpl.java:307)
	at stroom.importexport.impl.ContentResourceImpl.fetchDependencies(ContentResourceImpl.java:193)
	at stroom.importexport.impl.ContentResourceImpl.fetchDependencies(ContentResourceImpl.java:45)
	at jdk.internal.reflect.GeneratedMethodAccessor901.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory$1.invoke(ResourceMethodInvocationHandlerFactory.java:81)
```

**Comment by at055612** [comment]
Fixed in 7.0-beta.207
